samples/snippets/update_rollout_remove_default.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Game Servers sample for updating the rollout of a Game Server
+Deployment to remove the default config.
+
+Example usage:
+    python update_rollout_remove_default.py --project-id <project-id> --deployment-id <deployment-id>
+"""
+
+import argparse
+
+from google.cloud import gaming
+from google.cloud.gaming_v1.types import game_server_deployments
+from google.protobuf import field_mask_pb2 as field_mask  # type: ignore
+
+import wait_for_operation
+
+# [START gaming_update_rollout_remove_default]
+
+
+def update_rollout_remove_default(project_id, deployment_id):
+    """Creates a game server deployment."""
+
+    client = gaming.GameServerDeploymentsServiceClient()
+
+    # Location is hard coded as global, as Game Server Deployments can
+    # only be created in global.  This is done for all operations on
+    # Game Server Deployments, as well as for its child resource types.
+    request = game_server_deployments.UpdateGameServerDeploymentRolloutRequest()
+    request.rollout.name = f"projects/{project_id}/locations/global/gameServerDeployments/{deployment_id}"
+    request.rollout.default_game_server_config = ""
```

Comment:
Does this need to be explicitly set to an empty string?

Reply:
Yes, this is to remove the default config so the field needs to be set to an empty string.